packages/components/src/overlay/tests/jest/usePopup.test.tsx

```diff
@@ -455,8 +455,8 @@ test("closing the popup with esc keypress return the focus to the trigger", asyn
         />
     );
 
-    act(() => {
-        userEvent.click(getByTestId("trigger"));
+    await act(() => {
```

Comment:
all other lines have the format: 

```
    await act(() => userEvent.click(getByTestId("trigger")););
```

no need for the brackets and the return